File: src/main/java/com/travel/domain/travel/domain/city/TravelCityRepository.java
package com.travel.domain.travel.domain.city;

import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface TravelCityRepository extends JpaRepository<TravelCity, Long> {
    List<TravelCity> findByTravelId(Long travelId);
    List<TravelCity> findByCityId(Long cityId);
}

File: src/main/kotlin/com/travel/domain/travel/domain/city/TravelCityRepository.kt
package com.travel.domain.travel.domain.city

import org.springframework.data.jpa.repository.JpaRepository

interface TravelCityRepository: JpaRepository<TravelCity, Long> {
    fun findByTravelId(travelId: Long): List<TravelCity>
    fun findByCityId(cityId: Long): List<TravelCity>
}